pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -80,10 +73,30 @@ func checkKvmEnabled() error {
 
 func fixKvmEnabled() error {
 	logging.Debug("Trying to load kvm module")
-	stdOut, stdErr, err := crcos.RunWithPrivilege("Load kvm kernel module","modprobe", "kvm")
+	stdOut, stdErr, err := crcos.RunWithPrivilege("Load kvm kernel module", "modprobe", "kvm")
```

Comment:
Can you move this to the initial commit, since this is when the missing space was introduced.